packages/react-form-renderer/src/form-renderer/form-renderer.js

```diff
@@ -144,13 +175,13 @@ const FormRenderer = ({
               ffGetRegisteredFields: form.getRegisteredFields,
               getRegisteredFields: internalGetRegisteredFields,
               initialValues: props.initialValues,
+              schema,
             },
           }}
         >
           {FormTemplate && <FormTemplate formFields={formFields} schema={schema} {...FormTemplateProps} />}
 
-          {isFunc(children) && children({ formFields, schema })}
-          {typeof children === 'object' && cloneElement(children, { formFields, schema })}
+          {renderChildren(children, { formFields, schema })}
```

Comment:
could we get something like:

`children && render(...` 

It will remove the calling of the function and from the function the null/undefined can be removed.